datalad_crawler/nodes/annex.py

```diff
@@ -44,6 +44,8 @@
 from datalad.support.exceptions import AnnexBatchCommandError
 from datalad.support.network import get_url_straight_filename
 from datalad.support.network import get_url_disposition_filename
+# legacy import from -core, not longer part of the Repo classes
+from datalad.tests.utils import get_branch_commits
```

Comment:
all these changes as is would make datalad-crawler dependent on unreleased datalad.  I guess an adapter needs to be placed/used in datalad_crawler/utils.py to make it compatible